dawn/src/dawn/Compiler/DawnCompiler.h

```diff
@@ -20,12 +20,31 @@
 #include "dawn/Optimizer/OptimizerContext.h"
 #include "dawn/Support/DiagnosticsEngine.h"
 #include "dawn/Support/NonCopyable.h"
+#include <list>
 #include <memory>
 
 namespace dawn {
 
 struct SIR;
 
+/// @brief Enumeration of all pass groups
+enum class PassGroup {
+  Parallel,
```

Comment:
what is the order of these?

Reply:
The ordering isn't defined by the enumeration, although that is the order they were initially called in. See `defaultPassGroups()`.